src/Tests/Tests.MonoTorrent.IntegrationTests/IntegrationTests.cs

```diff
@@ -250,11 +247,18 @@ private void OnHttpContext (IAsyncResult ar)
             if (_failHttpRequest) {
                 _failHttpRequest = false;
                 ctx.Response.StatusCode = 500;
-            } else if (localPath.Contains (relativeSeedingPath)) {
+                ctx.Response.Close ();
+            } else if (!localPath.Contains (relativeSeedingPath)) {
+                ctx.Response.StatusCode = 404;
+                ctx.Response.Close ();
```

Comment:
@borigas The missing `Response.Close` call is why the `Retry` test needed a much longer execute.